### ISSUE TITLE
create DocTemplates for main webapp components

### ISSUE THREAD

**Issue by georgebusby** [wtchg-kwiatkowski/observatory-web#356]
The following components are usually hard-coded into pages. Suggestion is to put them into templates, with  the functionality to pass parameters using the e.g. <DocTemplate path="XXX" drug_id="YYY> syntax. These should then be used throughout webapp and in articles.

- proportion plots
- hirstian plots
- maps

**Comment by leehart** [comment]
These templates might all be best kept in their own sub-directory, maybe called "articles" or something, to highlight the dependent-relationship that the articles have with them. It might be prudent to keep them separate / not get them mixed up with other templates, even though they might be very similar.

**Comment by benjeffery** [comment]
I agree with Lee on the subfolder. I think these templates should be made on-demand though,  I'd like to rescope this to just making those needed for the current blog pages. 

There is no need to migrate the existing doc pages. They can use these new fragments when and as they get updated.
